sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/ExposureDetailsService.java

```diff
@@ -0,0 +1,61 @@
+package org.sormas.e2etests.services;
+
+import com.github.javafaker.Faker;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import javax.inject.Inject;
+import org.sormas.e2etests.enums.ActivityTypes;
+import org.sormas.e2etests.enums.Roles;
+import org.sormas.e2etests.enums.YesNoUnknownOptions;
+import org.sormas.e2etests.pojo.web.ExposureDetails;
+
+public class ExposureDetailsService {
+
+  private final Faker faker;
+
+  @Inject
+  public ExposureDetailsService(Faker faker) {
+    this.faker = faker;
+  }
+
+  public ExposureDetails buildInputExposureDetails() {
+    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MM/dd/yyyy");
+    return ExposureDetails.builder()
+        .startOfExposure(formatter.format(LocalDate.now().minusDays(5)))
+        .endOfExposure(formatter.format(LocalDate.now().minusDays(2)))
+        .exposureDescription(faker.medical().symptoms())
+        .typeOfActivity(ActivityTypes.VISIT.toString())
+        .exposureDetailsRole(Roles.VISITOR.toString())
+        .riskArea(YesNoUnknownOptions.YES.toString())
+        .indoors(YesNoUnknownOptions.YES.toString())
+        .outdoors(YesNoUnknownOptions.YES.toString())
+        .wearingMask(YesNoUnknownOptions.YES.toString())
+        .wearingPpe(YesNoUnknownOptions.YES.toString())
+        .otherProtectiveMeasures(YesNoUnknownOptions.YES.toString())
+        .shortDistance(YesNoUnknownOptions.YES.toString())
+        .longFaceToFaceContact(YesNoUnknownOptions.YES.toString())
+        .animalMarket(YesNoUnknownOptions.YES.toString())
+        .percutaneous(YesNoUnknownOptions.YES.toString())
+        .contactToBodyFluids(YesNoUnknownOptions.YES.toString())
+        .handlingSamples(YesNoUnknownOptions.YES.toString())
+        .contactToSourceCase("")
```

Comment:
why is this value empty?

Reply:
Based on the previous selections, they are empty. As i tried to use real lifelike dataand since there is no business rule in scope for the test I assumed is ok here to leave it like this and just go with them empty.